### PR TITLE
added notes on dual wielding melee weapons

### PR DESCRIPTION
**Pull request by bleehu**
modified:   03_Feats.txt

### CODE REVIEW

**Comment by bleehu** [comment]
Should fix Issue #514

File: CharacterCreation/03_Feats.txt [hunk 1]
```diff
@@ -149,12 +149,31 @@ Prerequisites:
...
+or one-handed melee weapons, etc. Additionally the sum of the strength requirements 
+must be equal to or less than your Carry Strength. While Dual Weilding, you cannot 
+take the grapple Action nor the Disarm Action. Any time you would attack with a 
+reaction or if an ability of yours or an ally's would give you a free attack, you
+may only attack with one of your dual wielded weapons, but you may pick which one
+to use. 
+
+	While wielding two guns, you may fire both at the same time at the same target 
+with a +3 miss chance for each weapon. You may not target more than one target at 
+a time. Reloading while dual wielding adds 20% to the difficulty of the reload.
```
Comment (rosebrallier) [inline CharacterCreation/03_Feats.txt]: Can we nix the % while we're here?

**Review by rosebrallier — APPROVED** [review]
Shipit!

File: CharacterCreation/03_Feats.txt [hunk 2]
```diff
@@ -149,12 +149,31 @@ Prerequisites:
...
+as normal in the same Action, with a -2 to your melee accuracy for each. If you 
+are attacked with a melee attack, you may only use one of your melee weapon's 
+accuracies as a bonus to your defense. You do get to pick which weapon you would
+like to use to defend though. 
+
+	In addition, this feat lets you take the 'Dual Guard' Action as the last 
+Action of your turn so long as you are weilding two melee weapons. If you do so, 
+you spend a whole turn "putting up your dukes" and preparing to block attacks 
+with both weapons. Mechanically, this gives you the guard bonus from both weapons
+as well as an extra +2 to your guard DC for having two weapons to block with.
```
Comment (Turtlelord26) [inline CharacterCreation/03_Feats.txt:176]: nb. Extra +2 on top of the second weapon's Guard bonus seems like overkill, balance-wise. I would like it to be watched carefully.

Comment (Turtlelord26) [reply CharacterCreation/03_Feats.txt:176]: Also it looks like we need some rules for offhand shields.

**Comment by bleehu** [comment]
I'm expecting the actual PUTLs to have deminishing marginal returns, but for the UTLs to increase linearly; thereby creating cheap Utility with little actual gain in power.

**Pull request merged**